pkg/cmd/server/start/start_master.go

```diff
@@ -378,7 +380,7 @@ func (m *Master) Start() error {
 	}
 
 	if m.api {
-		glog.Infof("Starting master on %s (%s)", m.config.ServingInfo.BindAddress, version.Get().String())
+		glog.Infof(MasterStartupMessage, m.config.ServingInfo.BindAddress, version.Get().String())
```

Comment:
@liggitt is this too invasive? Pretty much all the log messages diagnostics looks for are stuffed directly into glog so I can only get at them by pulling them out into exported constants (which assumes they're even in code we control).

However this is pretty effective for testing the relevance and correctness of the log matchers.